### ISSUE TITLE
How to reduce texture of images? 

### ISSUE THREAD

**Issue by Xinxiang7** [SystemErrorWang/White-box-Cartoonization#18]
The comparison between my result and author's:
mine_1
![87](https://user-images.githubusercontent.com/24989537/85936840-efbe0d00-b930-11ea-93e5-143b4ed65a0d.jpg)
demo_1
![87](https://user-images.githubusercontent.com/24989537/85936847-f8164800-b930-11ea-8450-96c5d62eadb8.jpg)
mine_2
![longdog1](https://user-images.githubusercontent.com/24989537/85936857-154b1680-b931-11ea-8464-aaa2b79132b6.jpg)
demo_2
![longdog1](https://user-images.githubusercontent.com/24989537/85936860-1c722480-b931-11ea-9e65-3578dfac5f09.jpg)
The texture is so obvious.   
I try different methods:
firstly, I increase weight of different losses respectively, but the result doesn't change too much or even worse. And then I only train the scenery, the result becomes much worse. After that I change the lg_loss according to ### **Least Squares Generative Adversarial Networks**:
`d_loss = 0.5*(tf.reduce_mean((real_logit - 1)**2) + tf.reduce_mean(fake_logit**2))`
`# some parameters need to be change according to Least Squares Generative Adversarial Networks
 d_loss = 0.5*(tf.reduce_mean((real_logit - 1)**2) + 0.5*tf.reduce_mean(fake_logit**2))`
But, the result doesn't change.
What's more, I check the tensorboard. and find out that both the g_loss_blur and g_loss_gray don't change much.  I know that the loss function has much relationship with that. I read some papers such as Martin Arjovsky's ### **Wassertein GAN**, ### **Improved Training of Wasserstein GANs** and ### **Spectral Normalization for Generative Adversarial Networks**. 
But I still don't know how to decrease g_loss_blur and g_loss_gray during training. I can't find the reason. Anyone doing the training would like to do me a favor? Thanks very much.

**Comment by SystemErrorWang** [comment]
It's normal that the g_loss_blur and the g_loss_gray is not decreasing during training, as the goal of GAN is to balance between G and D and force them to reach Nash equilibrium. If you want to decreas the texture, you can decrease the weight of g_loss_gray.

**Comment by Xinxiang7** [comment]
@SystemErrorWang Thanks very much for your reply. I will try.
